src/pageLayout/constants/navigationHierarchy.ts

```diff
@@ -33,11 +36,10 @@ export interface NavItem {
   activeKeywords: string[]
 }
 
-export const generateNavItems = (): NavItem[] => {
-  const state: AppState = getStore().getState()
-  const orgID = state?.resources?.orgs?.org?.id ?? ''
-  const quartzMe = state?.me?.quartzMe ?? null
-
```

Comment:
Ideally we should not [be calling getStore()](https://stackoverflow.com/questions/34455996/is-using-getstate-in-redux-an-anti-pattern) in a utility function if we can help it (which we can here).